src/test/groovy/org/shipkit/notes/improvements/CommaSeparatedTest.groovy

```diff
@@ -1,15 +0,0 @@
-package org.shipkit.notes.improvements
-
-import spock.lang.Specification
-
-class CommaSeparatedTest extends Specification {
```

Comment:
I'm wondering if we even need CommaSeparated class given that we can use org.shipkit.internal.gradle.util.StringUtil#join